core/src/main/java/nu/yona/server/messaging/service/MessageDto.java

```diff
@@ -186,9 +186,7 @@ protected final SenderInfo getSenderInfo(UserDto actingUser, Message messageEnti
 					return createSenderInfoForSelf(actingUser);
 				}
 
-				// Note: actingUser may be out of sync here, because the message action may have changed its state
```

Comment:
Because it is now no longer the case, it is synced in the message action handlers before going here. I can still leave a warning if you like?